src/pkg/services/m365/m365.go

```diff
@@ -86,6 +86,34 @@ func UserPNs(ctx context.Context, acct account.Account, errs *fault.Bus) ([]stri
 	return ret, nil
 }
 
+type Site struct {
+	// URL that displays the item in the browser
+	URL string
```

Comment:
nit: I'm being overly picky about naming consistency.
```suggestion
	WebURL string
```

Reply:
I appreciate nits always! I will change it, thank you.